ci/infra/testrunner/testrunner.py

```diff
@@ -264,6 +268,9 @@ def main():
     cmd_ssh = commands.add_parser("ssh", parents=[node_args, ssh_args], help="Execute command in node via ssh.")
     cmd_ssh.set_defaults(func=ssh)
 
+    crio_mirror_command = commands.add_parser("crio_mirror", help="Set mirror registry for CI testing in the workers")
```

Comment:
Isn't this command overlapping with `--registry-mirror`? aren't they doing kind of the same thing? I am confused about how they differ.

Reply:
interesting, I didnt see that at all...let me check!

Reply:
OMG! that would work! Good catch!